easybuild/easyconfigs/g/GEOS/GEOS-3.12.2-GCC-13.3.0.eb

```diff
@@ -0,0 +1,22 @@
+easyblock = 'CMakeMake'
+
+name = 'GEOS'
+version = '3.12.2'
+
+homepage = 'https://trac.osgeo.org/geos'
+description = """GEOS (Geometry Engine - Open Source) is a C++ port of the Java Topology Suite (JTS)"""
+
+toolchain = {'name': 'GCC', 'version': '13.3.0'}
+
+source_urls = ['https://download.osgeo.org/geos/']
+sources = [SOURCELOWER_TAR_BZ2]
+checksums = ['34c7770bf0090ee88488af98767d08e779f124fa33437e0aabec8abd4609fec6']
+
+builddependencies = [('CMake', '3.29.3')]
```

Comment:
@ankekreuzer Existing recent `GEOS` easyconfigs also have this, we should retain that?

```python
# Build static and shared libraries (2/2)
configopts = ['', '-DBUILD_SHARED_LIBS=OFF']
```

as well as:

```python
toolchain = ...
toolchainopts = {'pic': True}
```

Did you run into trouble with this somehow?

Reply:
No, it looks like our version at JSC didn't had this. I'll add it here and also in our repo at JSC (we try keep as close as possible to "official" repo). Thanks for spotting this!